packages/goblint-cil/goblint-cil.1.7.3/opam

```diff
@@ -0,0 +1,29 @@
+opam-version: "1.2"
```

Comment:
I think you also need a `available : [ ocaml-version >= "4.02.3" ]` at least, due to the use of Bytes

```
# File "src/formatlex.mll", line 148, characters 13-23:
# Error: Unbound module Bytes
# Command exited with code 2.
```